projects/kubernetes-sigs/cluster-api/patches/0003-Unstacked-etcd-and-controlplane-upgrade.patch

```diff
@@ -63,21 +63,21 @@ index 58ef4a74e..1c7d411bd 100644
  
  // MachineAddressType describes a valid MachineAddress type.
 diff --git a/api/v1beta1/common_types.go b/api/v1beta1/common_types.go
-index 7d19c0e05..f17d3b066 100644
+index 1d8d0d633..b1ffbd135 100644
 --- a/api/v1beta1/common_types.go
 +++ b/api/v1beta1/common_types.go
-@@ -112,6 +112,9 @@ const (
- 	// An external controller must fulfill the contract of the InfraCluster resource.
- 	// External infrastructure providers should ensure that the annotation, once set, cannot be removed.
- 	ManagedByAnnotation = "cluster.x-k8s.io/managed-by"
```

Comment:
We are removing this annotation?

Reply:
So these are just the context lines for the diff, they're not getting modified by the patch. Between 1.1.3 and 1.2.0, they had added the `TopologyDryRunAnnotation`, that now acts as the context lines, replacing this one. The actual patch is line 73 to 75, where we add the `ControlPlaneUpgradeCompletedAnnotation`.

Reply:
Oh right I forgot, thanks for that